### DISCUSSION TITLE
Hotreload

### DISCUSSION THREAD

**Discussion by vtmx** [tsujan/Kvantum#585] [Q&A]
Hello tsujan, I'm Vitor from Brazil. I am a beginner in KDE, I migrated precisely because of the flexibility of the interface and being able to customize my own.

My current goal is to create a theme based on One Dark colors, I started my learning process by editing the standard colors of Breeze. This was the result:
https://raw.githubusercontent.com/vtmx/kde-onedark-color/main/screenshot.png

In the middle of the learning process, I realized how limited Breeze was in relation to some elements of the GUI, mainly in hover and active events, where I could only change the border color for one and only the background for another.

It was then that I discovered Kvantum, I was impressed with the range of possibilities that it would give me if I learned to use it. When I was editing my theme.colors, I switched from the previous theme to the current one to activate the current color modifications, etc. It was a process that took a long time.

Is there a HotReload on Kvantum?

I researched some videos about creating themes on Kvantum, but I didn't find anything about it. I think a channel is missing on the subject.

Congratulations on this excellent system, I wish you success and health.

**Comment by tsujan** [comment]
> I am a beginner in KDE, I migrated precisely because of the flexibility of the interface and being able to customize my own.

To prevent a probable misunderstanding, I should emphasize that Kvantum isn't a KDE program. It's tuned to KDE and LXQt but can be used to style Qt apps everywhere, even under Gnome. It's definitely independent of KDE color schemes but matching color schemes may be needed under KDE because of KDE problems.

> Is there a HotReload on Kvantum?

I don't know what you mean by "HotReload"; haven't heard of it.

> I researched some videos about creating themes on Kvantum

Kvantum has a comprehensive documentation inside its source. Making new themes needs patience and isn't easy at first but, like any skill, gets easier with more practice. You could start with an existing theme and change it by consulting Kvantum's documentation.

Thanks for the kind words!

**Reply by vtmx** [reply]
About the hot reload. Currently with each change in the SVG file, I then click on the preview button. Is there any way to leave the preview window open and as I save .kvconfig or .svg it will be updated with each save?

**Reply by tsujan** [reply]
There's no way to do so directly because those files aren't edited by Kvantum Manager (e.g., Inkscape is a separate app). Theoretically, it's possible to write a code to "watch" the files; practically, it would have serious side effects. So, no.

**Reply by vtmx** [reply]
Ok.
